pkg/recommend/advisor/ehpa.go

```diff
@@ -96,11 +96,11 @@ func (a *EHPAAdvisor) Advise(proposed *types.ProposedRecommendation) error {
 	targetCpuUtilization := int32(50) // todo: configurable
 	maxReplicasFactor := 1.2          // todo: configurable
 
-	maxCpuUtilization := int32(int64(maxCpuUsage) * 1000 * 100 / requestTotal)
+	maxCpuUtilization := int32(int64(maxCpuUsage) * 1000 * 100 * int64(len(a.Pods)) / requestTotal)
```

Comment:
why multiply pods num. maxCpuUsage is average max usage of each pod?  I see it is computed only by one timeseries tsList[0], not all

Reply:
requestTotal contains total request for workload, so we need to add this.

Reply:
maxCpuUsage is max of workload‘s(sum of all pods) usage too, so the original code is right, do not need fix